examples/layers/preprocessing/random_shear_bbox_demo.py

```diff
@@ -0,0 +1,100 @@
+# Copyright 2022 The KerasCV Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+"""random_shear_bboxdemo.py shows how to use the RandomShear preprocessing layer
+   for object detection.
+Operates on the voc dataset.  In this script the images and bboxes
+are loaded, then are passed through the preprocessing layers.
+Finally, they are shown using matplotlib.
+"""
+import matplotlib.pyplot as plt
+import numpy as np
+import tensorflow as tf
+import tensorflow_datasets as tfds
+
+from keras_cv.bounding_box import convert_format
+from keras_cv.layers import preprocessing
+
+IMG_SIZE = (256, 256)
+BATCH_SIZE = 9
+
+
+def resize(inputs):
+    """mapping function to create batched image and bbox coordinates"""
+    inputs["image"] = tf.image.resize(inputs["image"], IMG_SIZE)[0]
+    inputs["objects"]["bbox"] = tf.squeeze(
+        convert_format(
```

Comment:
After I merge https://github.com/keras-team/keras-cv/pull/556 you can rely on unbatched preprocessing!

Reply:
merged